src/libvx.c

```diff
@@ -973,6 +1125,11 @@ static vx_error vx_decode_frame(vx_video* me, static AVFrame* out_frame_buffer[5
 		goto cleanup;
 	}
 
+	if (packet->stream_index == me->audio_stream && !me->swr_ctx) {
```

Comment:
Don't process audio frames unless audio is enabled. In a follow-up this could be extended to only initialize the audio codec if it is actually needed.